modules/openapi-generator/src/main/java/org/openapitools/codegen/languages/TypeScriptAngularClientCodegen.java

```diff
@@ -86,6 +86,8 @@ public TypeScriptAngularClientCodegen() {
         apiPackage = "api";
         modelPackage = "model";
 
+        typeMapping.put("DateTime", "Date");
```

Comment:
I think it would be good to remove this, i.e. to change the mapping from `Date` to `string` for `format: date-time`, since the typescript-angular generator does not support serialization for the models.
Note: this would pose a problem, since serialization for query params is implemented, see e.g. https://github.com/OpenAPITools/openapi-generator/pull/5225

Reply:
Right, it was unclear to me whether de-serialization was supported for angular. And it would require changing the unit tests.

I'm tempted to just keep the current behavior and punt on "fixing" it for angular.

Reply:
@macjohnny we can remove it separately (for angular and other generators except fetch/reduxquery), since it **might** break existing behavior for current users

Reply:
Ok, I updated the rest of the generators to keep the output consistent.